### DISCUSSION TITLE
getLicense is not getting called for DRM content. I am using 6.0.0-rc.0.

### DISCUSSION THREAD

**Discussion by rahulnainwalttn** [TheWidlarzGroup/react-native-video#3724] [Q&A]
### What are you trying to do?

I am trying to play DRM content in iOS but it is failing with `The operation couldn't be completed` error. When I tried to debug it, found `getLicense` callback is not getting called.

Also, can someone confirm which are required fields for `getLicense` callback to trigger?

### What have you tried?

Configuring DRM with FairPlay and a `getLicense` callback (see code below), but the `getLicense` callback is never invoked and playback fails with `The operation couldn't be completed`.

### Relevant code

```tsx
drm: {
  type: DRMType.FAIRPLAY,
  certificateUrl: 'https://*****',
  getLicense: (spcString) => {
    console.log('klklskl ', spcString);
  },
},
```

### Version

v6

### Platforms

iOS

**Comment by freeboub** [comment]
We don't have such feedback for now ...

**Reply by rahulnainwalttn** [reply]
Yes, I was testing it in the simulator. In a real device, it is working fine.
